var/spack/repos/builtin/packages/py-wandb/package.py

```diff
@@ -10,23 +10,48 @@ class PyWandb(PythonPackage):
     """A tool for visualizing and tracking your machine
     learning experiments."""
 
-    homepage = "https://github.com/wandb/"
-    url = "https://github.com/wandb/client/archive/v0.10.1.tar.gz"
+    homepage = "https://github.com/wandb/wandb"
+    pypi = "wandb/wandb-0.13.9.tar.gz"
 
+    maintainers = ["thomas-bouvier"]
+
+    version("0.13.9", sha256="0a17365ce1f18306ce7a7f16b943094fac7284bb85f4e52c0685705602f9e307")
     version("0.10.1", sha256="abd334cd1460ac1f6e5aa959d3e04c46cd246f96cfc3323fc0572916760d32ab")
```

Comment:
```suggestion
    version("0.10.1", sha256="d02427cda58a6618ba10a027a76d9e3f68ad923d35964b1b68785c49e5160009", deprecated=True)
```
PyPI tarballs have different checksums than GitHub. Also, the 0.10.1 release was yanked from PyPI suggesting it contains a serious bug, so we should probably deprecate it.